### DISCUSSION TITLE
Shell Shortcut

### DISCUSSION THREAD

**Discussion by meteorquake** [sumatrapdfreader/sumatrapdf#4353] [Q&A]
Is there a way in Sumatra to execute an .exe from a shortcut, using parameters for filename and page (e.g. %1 and %p) like you have with ExternalViewers?
Cheers, David

**Comment by kjk** [comment]
It's not currently possible but if I implement https://github.com/sumatrapdfreader/sumatrapdf/issues/4348, it'll be

**Comment by meteorquake** [comment]
That would be great, I would use it well, I was envisaging a command that could be fitted into a single line, but user-defined command names sound good and you'll have chosen that route for good reasons, without I'm sure prejudicing existing capabilities.
